structuredevent-service-cdp/src/main/java/com/sequenceiq/cloudbreak/structuredevent/service/telemetry/mapper/FreeIpaUseCaseMapper.java

```diff
@@ -79,6 +81,11 @@ private UsageProto.CDPFreeIPAStatus.Value lastStepToUseCaseMapping(String rootFl
                             UsageProto.CDPFreeIPAStatus.Value.UPGRADE_FINISHED,
                             UsageProto.CDPFreeIPAStatus.Value.UPGRADE_FAILED);
                     break;
+                case "UpgradeCcmFlowEventChainFactory":
+                    useCase = getFreeIpaStatus(nextFlowState, "UPGRADE_CCM_FINISHED_STATE_NAME",
```

Comment:
Sanity check: was the intention here to create a CCM_UPGRADE_STARTED event when the UpgradeCcmFlow starts in the UpgradeCcmFlowEventChain and have a CCM_UPGRADE_FINISHED event when the UpgradeCcmFlow finishes and not when the whole flowchain finishes (so when UpdateUserDataFlow finishes)?

Reply:
done